### PR TITLE
Student scheduling

### PR DESCRIPTION
**Pull request by CameronBourque**
Some preliminary code merging the student controller with the user controller. It's only been partially tested but it should do for now.

### CODE REVIEW

File: app/views/users/schedule_student.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      <% date = session.scheduled_datetime.localtime.strftime("%B %d at %l:%M %p %Z") %>
+      <td><%= date %></td>
+      <!--
+        TODO: Pass user that was passed in to button instead of hard coded value
+            ex: {:action => 'schedule', :id => session.id, :user => params[:user]}
+      -->
+      <td><%= button_to "Join session", {:action => 'schedule_session_student', :sessionID => session}, :method => :post %></td>
+    </tr>
+  <% end %>
+</table>
```
Comment (baileyjrob) [inline app/views/users/schedule_student.html.erb]: Make sure to add a newline at the end of the file, as-per coding standards

**Review by baileyjrob — CHANGES_REQUESTED** [review]
Are you sure that everything works? I'm a little hesitant to merge such a big architectural change halfway through implementation

File: app/views/users/schedule_student.html.erb (hunk 1 above)
Comment (CameronBourque) [reply app/views/users/schedule_student.html.erb]: Added the newline in the newest commit

**Comment by CameronBourque** [comment]
It does work but I haven't made rspec tests for it so there's a small chance for bugs but they shouldn't affect conventional use. This should be enough for people to work off of.

**Pull request merged**